### ISSUE TITLE
RequestTableTest throws warning on mocking method

### ISSUE THREAD

**Issue by othercorey** [cakephp/debug_kit#769]
```
1) DebugKit\Test\TestCase\Model\Table\RequestTableTest::testGc
Trying to set mock method "shouldGc" with addMethods(), but it exists in class "DebugKit\Model\Table\RequestsTable". Use onlyMethods() for methods that exist in the class.
```

**Comment by ADmad** [comment]
Sound related to https://github.com/cakephp/cakephp/pull/14792. So the error should go away after next release.

**Comment by othercorey** [comment]
I couldn’t remember if anything changed for 4.1 so I logged it here.

**Comment by othercorey** [comment]
Fixed in cakephp 4.1.
